### ISSUE TITLE
Rule blocked: CORERULES-631 (SEND-190.1)

### ISSUE THREAD

**Issue by JozefAerts** [cdisc-org/cdisc-rules-engine#558]
For rule SEND-190.1: When OMTESTCD=OWBR  or OMTESTCD=OWBW or OMTESTCD=OWHT, then OMORRESU="RATIO" or ORMORRESU="%"
we probably need a new or extended function like: 
- name: OMSTRESU
  operator: has_different_values
  within:
    - OMTESTCD
    
as e.g.:
Check:
  all:
    - name: OMTESTCD
      operator: equal_to
      value: 'OWHT'
    - name: OMSTRESU
      operator: has_different_values
doesn't seem to filter on the rows with OMTESTCD=OWHT for checking with has_different_values.

A test file is attached.
[unit-test-coreid-SENDIG190-1-positive.xlsx](https://github.com/cdisc-org/cdisc-rules-engine/files/12839981/unit-test-coreid-SENDIG190-1-positive.xlsx)

**Comment by gerrycampion** [comment]
@JozefAerts Does [is_inconsistent_across_dataset](https://cdisc-org.github.io/conformance-rules-editor/#/check_operator?id=is_inconsistent_across_dataset) not do this already? Or is that one not filtering properly. It seems we have these two operators that do close to the same thing.

Also, I checked your implementation for `CDISC.SENDIG.190.1B`. Is this implementation working? Can the ticket be closed?

**Comment by JozefAerts** [comment]
Thanks @gerrycampion! I will try first using "is_inconsistent_across_dataset"

**Comment by JozefAerts** [comment]
With "is_inconsistent_across_dataset" seems to work correctly, but it is a bit weird how the violating row numbers are reported.
Will discuss this afternoon during "office hours".

**Comment by JozefAerts** [comment]
@gerrycampion Yes, "is_inconsistent_across_dataset" does the job, so you may close the ticket.
Sam explained today during the "office hours" that the reporting of the rows violating the rule is based on "majority" versus "minority" of the values within the group, and not on "relative to the first in the group".
So, I will add a few more tests for SEND-190.1 to figure this out.
As this is something else though than this ticket, I propose to close the current one.
